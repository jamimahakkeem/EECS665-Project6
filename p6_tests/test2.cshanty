void main() {
	bool b;
	b = true;
	b = false;
	b = !b;
	b = b && b;
	b = b || b;	
}