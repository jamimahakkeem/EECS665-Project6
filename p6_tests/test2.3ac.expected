[BEGIN GLOBALS]
[END GLOBALS]
[BEGIN main LOCALS]
b (local var of 8 bytes)
tmp0 (tmp var of 8 bytes)
tmp1 (tmp var of 8 bytes)
tmp2 (tmp var of 8 bytes)
[END main LOCALS]
main:       enter main
            [b] := 1
            [b] := 0
            [tmp0] := NOT8 [b]
            [b] := [tmp0]
            [tmp1] := [b] AND8 [b]
            [b] := [tmp1]
            [tmp2] := [b] OR8 [b]
            [b] := [tmp2]
lbl_0:      leave main
